app/assets/javascripts/controllers/StudentsIndexCtrl.js

```diff
@@ -51,5 +51,26 @@ planner.controller('StudentsIndexCtrl', ['$scope', 'Restangular', 'advisors', 's
       });
     };
 
+    $scope.createStudent = function() {
+      var newStudent = Restangular.restangularizeElement(null, $scope.newStudent, 'students');
+      newStudent.post()
+        .then(function() {
```

Comment:
No need to restangularize the student. Just post: `Restangular.all('students').post(params).then.....`
Note for refactoring